### PR TITLE
network/cni: more informative log messages

### PR DESCRIPTION
**Pull request by dcbw**
Log details about the pod name, namespace, sandbox ID, and result.
Remove some useless messages that are now redundant.

@freehan @thockin @squeed @bowei 

```release-note
NONE
```

### CODE REVIEW

**Review by squeed — APPROVED** [review]

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 1]
```diff
@@ -321,22 +320,21 @@ func (plugin *cniNetworkPlugin) deleteFromNetwork(network *cniNetwork, podName s
...
+	pdesc := podDesc(podNamespace, podName, podSandboxID)
 	netConf, cniNet := network.NetworkConfig, network.CNIConfig
-	glog.V(4).Infof("About to del CNI network %v (type=%v)", netConf.Name, netConf.Plugins[0].Network.Type)
+	glog.V(4).Infof("Deleting %s from network %s/%s netns %q", pdesc, netConf.Plugins[0].Network.Type, netConf.Name, podNetnsPath)
 	err = cniNet.DelNetworkList(netConf, rt)
 	// The pod may not get deleted successfully at the first time.
 	// Ignore "no such file or directory" error in case the network has already been deleted in previous attempts.
 	if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
-		glog.Errorf("Error deleting network: %v", err)
+		glog.Errorf("Error deleting %s from network %s: %v", pdesc, netConf.Name, err)
```
Comment (freehan) [inline pkg/kubelet/dockershim/network/cni/cni.go]: Probably need to be consistent with above. `network %s/%s`

File: pkg/kubelet/dockershim/network/cni/cni.go [hunk 2]
```diff
@@ -296,21 +290,26 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
+func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
```
Comment (freehan) [inline pkg/kubelet/dockershim/network/cni/cni.go]: kubelet has a util function that does similar stuff. But probably we should not import it into this package?

**Review by freehan — COMMENTED** [review]
just nits

File: pkg/kubelet/dockershim/network/cni/cni.go (hunk 2 above)
Comment (dcbw) [reply pkg/kubelet/dockershim/network/cni/cni.go]: @freehan I did see PodFullName() but that doesn't do the containerID too.  Which one were you thinking of?

Comment (freehan) [reply pkg/kubelet/dockershim/network/cni/cni.go]: That was the one I am looking at. Never mind then.

**Comment by dcbw** [comment]
@freehan PTAL, thanks!

File: pkg/kubelet/dockershim/network/cni/cni.go (hunk 1 above)
Comment (dcbw) [reply pkg/kubelet/dockershim/network/cni/cni.go]: Done.

**Pull request merged**